ffi/SDL2_0.lua

```diff
@@ -68,7 +68,7 @@ function S.open()
     if SDL.SDL_Init(bit.bor(SDL.SDL_INIT_VIDEO,
                             SDL.SDL_INIT_JOYSTICK,
                             SDL.SDL_INIT_GAMECONTROLLER)) ~= 0 then
-        error("Cannot initialize SDL.")
+        error("Cannot initialize SDL: " .. ffi.string(SDL.SDL_GetError()))
```

Comment:
Fun fact: I added the FFI header for it but I don't recall what I debugged with it. (Definitely not this line, 'cause I wouldn't have cleaned that out.)